swarm/network/stream/stream.go

```diff
@@ -68,6 +68,7 @@ type RegistryOptions struct {
 	SkipCheck       bool
 	DoSync          bool
 	DoRetrieve      bool
+	DoServeRetrieve bool
```

Comment:
To the best of my understanding - this value should be `true` by default (but unassigned bool is `false`), however I don't see an explicit fallback assignment (except for in the tests)